perception/probabilistic_occupancy_grid_map/schema/synchronized_grid_map_fusion_node.schema.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Parameters for Synchronized Grid Map Fusion Node",
+  "type": "object",
+  "properties": {
+    "ros__parameters": {
```

Comment:
The CI is complaining about the syntax in this file.

I think the properties should start with `/**` as required field, but it is now starting with `ros__parameters`